R/joint_optspace_helper.R

```diff
@@ -0,0 +1,105 @@
+#' Joint RPCA Ordination Across Multiple Compositional Tables
+#'
+#' Internal function that performs Robust PCA via joint OptSpace decomposition across multiple compositional tables.
+#' It splits each table into train/test sets, applies joint factorization, reconstructs sample and feature embeddings,
+#' optionally projects test samples, computes a sample distance matrix, and returns cross-validation error statistics.
+#'
+#' @param tables A list of compositional matrices or data frames with features as rows and samples as columns.
+#' @param n_components Number of principal components to compute.
+#' @param max_iterations Maximum number of optimization iterations for OptSpace.
+#' @param test_samples Character vector of sample IDs to be projected into the ordination space.
```

Comment:
Minor issue but due to internal pkg conventions the argument names should be with dot, like "n_components" -> "n.components" etc. Can you rename..?